restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/converter/AbstractConverter.java

```diff
@@ -663,4 +664,35 @@ protected void writeFaxCapability(final Boolean faxCapable, final JsonObject obj
             object.addProperty("fax_capable", Boolean.FALSE);
         }
     }
+
+    protected void writeError(final Error error, final HierarchicalStreamWriter writer) {
```

Comment:
If this is SMS-specific error, why did we create these methods in an generic converter?

Reply:
agree this converter is getting loaded with specific entity details that will be rarely reused on diff converters,move to specific converter instead